scheduler/scheduler_medium_test.go

```diff
@@ -285,7 +285,7 @@ func TestCreateTask(t *testing.T) {
 				// we are ok at this precision with being within 10% over the interval (10ms)
 				time.Sleep(interval * 110 / 100)
 				// check if the task is ended
-				So(tsk.State(), ShouldEqual, core.TaskEnded)
+				// So(tsk.State(), ShouldEqual, core.TaskEnded)
```

Comment:
I guess this test is disabled because it's flaky, but it shouldn't be disabled in this PR. I suggest to merge appropriate PR and uncomment this line.